LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties/letter-spacing.html

```diff
@@ -13,9 +13,28 @@
 <script>
 'use strict';
 
+function assert_is_equal_with_zero_special_handling(input, result) {
+  let is_zero_length = true;
+  if (input instanceof CSSMathSum) {
+    for (let operand of input.values)
+      is_zero_length &= (operand.value == 0);
```

Comment:
```suggestion
      is_zero_length &&= (operand.value == 0);
```

if you want to keep a boolean.

Reply:
Fixed.